2.0.0-alpha.7

matteobruni opened the pull request.

dependabot[bot] committed:
build(deps-dev): bump source-map-loader from 2.0.0 to 2.0.1

Bumps [source-map-loader](https://github.com/webpack-contrib/source-map-loader) from 2.0.0 to 2.0.1.
- [Release notes](https://github.com/webpack-contrib/source-map-loader/releases)
- [Changelog](https://github.com/webpack-contrib/source-map-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/source-map-loader/compare/v2.0.0...v2.0.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/mocha from 8.2.1 to 8.2.2

Bumps [@types/mocha](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/mocha) from 8.2.1 to 8.2.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/mocha)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 1.2.6 to 1.3.0

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 1.2.6 to 1.3.0.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/compare/svelte-check-1.2.6...svelte-check-1.3.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump @angular/router from 11.2.6 to 11.2.7

Bumps [@angular/router](https://github.com/angular/angular/tree/HEAD/packages/router) from 11.2.6 to 11.2.7.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/11.2.7/packages/router)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1432 from matteobruni/dependabot/npm_and_yarn/dev/source-map-loader-2.0.1

build(deps-dev): bump source-map-loader from 2.0.0 to 2.0.1

matteobruni committed:
Merge pull request #1433 from matteobruni/dependabot/npm_and_yarn/dev/types/mocha-8.2.2

build(deps-dev): bump @types/mocha from 8.2.1 to 8.2.2

matteobruni committed:
Merge pull request #1434 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-1.3.0

build(deps-dev): bump svelte-check from 1.2.6 to 1.3.0

matteobruni committed:
Merge pull request #1435 from matteobruni/dependabot/npm_and_yarn/dev/angular/router-11.2.7

build(deps): bump @angular/router from 11.2.6 to 11.2.7

dependabot[bot] committed:
build(deps-dev): bump @angular/compiler-cli from 11.2.6 to 11.2.7

Bumps [@angular/compiler-cli](https://github.com/angular/angular/tree/HEAD/packages/compiler-cli) from 11.2.6 to 11.2.7.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/11.2.7/packages/compiler-cli)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump preact-render-to-string from 5.1.16 to 5.1.18

Bumps [preact-render-to-string](https://github.com/developit/preact-render-to-string) from 5.1.16 to 5.1.18.
- [Release notes](https://github.com/developit/preact-render-to-string/releases)
- [Commits](https://github.com/developit/preact-render-to-string/compare/v5.1.16...5.1.18)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump jsoneditor from 9.1.7 to 9.2.0

Bumps [jsoneditor](https://github.com/josdejong/jsoneditor) from 9.1.7 to 9.2.0.
- [Release notes](https://github.com/josdejong/jsoneditor/releases)
- [Changelog](https://github.com/josdejong/jsoneditor/blob/develop/HISTORY.md)
- [Commits](https://github.com/josdejong/jsoneditor/compare/v9.1.7...v9.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump svelte from 3.36.0 to 3.37.0

Bumps [svelte](https://github.com/sveltejs/svelte) from 3.36.0 to 3.37.0.
- [Release notes](https://github.com/sveltejs/svelte/releases)
- [Changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte/compare/v3.36.0...v3.37.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1436 from matteobruni/dependabot/npm_and_yarn/dev/angular/compiler-cli-11.2.7

build(deps-dev): bump @angular/compiler-cli from 11.2.6 to 11.2.7

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-node-resolve from 11.2.0 to 11.2.1

Bumps [@rollup/plugin-node-resolve](https://github.com/rollup/plugins) from 11.2.0 to 11.2.1.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Commits](https://github.com/rollup/plugins/compare/node-resolve-v11.2.0...node-resolve-v11.2.1)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1437 from matteobruni/dependabot/npm_and_yarn/dev/preact-render-to-string-5.1.18

build(deps): bump preact-render-to-string from 5.1.16 to 5.1.18

matteobruni committed:
Merge pull request #1438 from matteobruni/dependabot/npm_and_yarn/dev/jsoneditor-9.2.0

build(deps-dev): bump jsoneditor from 9.1.7 to 9.2.0

matteobruni committed:
Merge pull request #1439 from matteobruni/dependabot/npm_and_yarn/dev/svelte-3.37.0

build(deps): bump svelte from 3.36.0 to 3.37.0

matteobruni committed:
Merge pull request #1440 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-node-resolve-11.2.1

build(deps-dev): bump @rollup/plugin-node-resolve from 11.2.0 to 11.2.1

dependabot[bot] committed:
build(deps-dev): bump chai from 4.3.3 to 4.3.4

Bumps [chai](https://github.com/chaijs/chai) from 4.3.3 to 4.3.4.
- [Release notes](https://github.com/chaijs/chai/releases)
- [Changelog](https://github.com/chaijs/chai/blob/main/History.md)
- [Commits](https://github.com/chaijs/chai/compare/4.3.3...v4.3.4)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump uglify-js from 3.13.2 to 3.13.3

Bumps [uglify-js](https://github.com/mishoo/UglifyJS) from 3.13.2 to 3.13.3.
- [Release notes](https://github.com/mishoo/UglifyJS/releases)
- [Commits](https://github.com/mishoo/UglifyJS/compare/v3.13.2...v3.13.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump bootstrap from 4.5.3 to 4.6.0

Bumps [bootstrap](https://github.com/twbs/bootstrap) from 4.5.3 to 4.6.0.
- [Release notes](https://github.com/twbs/bootstrap/releases)
- [Commits](https://github.com/twbs/bootstrap/compare/v4.5.3...v4.6.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1441 from matteobruni/dependabot/npm_and_yarn/dev/chai-4.3.4

build(deps-dev): bump chai from 4.3.3 to 4.3.4

matteobruni committed:
Merge pull request #1442 from matteobruni/dependabot/npm_and_yarn/dev/uglify-js-3.13.3

build(deps-dev): bump uglify-js from 3.13.2 to 3.13.3

matteobruni committed:
Merge pull request #1444 from matteobruni/dependabot/npm_and_yarn/dev/bootstrap-4.6.0

build(deps-dev): bump bootstrap from 4.5.3 to 4.6.0

matteobruni committed:
build: updates

matteobruni committed:
feat: added external presets to editor

dependabot[bot] committed:
build(deps-dev): bump ts-loader from 8.0.18 to 8.1.0

Bumps [ts-loader](https://github.com/TypeStrong/ts-loader) from 8.0.18 to 8.1.0.
- [Release notes](https://github.com/TypeStrong/ts-loader/releases)
- [Changelog](https://github.com/TypeStrong/ts-loader/blob/main/CHANGELOG.md)
- [Commits](https://github.com/TypeStrong/ts-loader/compare/v8.0.18...v8.1.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1446 from matteobruni/dependabot/npm_and_yarn/dev/ts-loader-8.1.0

build(deps-dev): bump ts-loader from 8.0.18 to 8.1.0

matteobruni committed:
build: removed empty element, it will be managed by end user

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.3
 - inferno-particles@1.7.3
 - jquery-particles@1.24.3
 - preact-particles@1.24.3
 - react-tsparticles@1.24.4
 - svelte-particles@1.7.3
 - particles.vue@2.7.3
 - particles.vue3@1.7.3
 - tsparticles-editor@1.24.3
 - tsparticles@1.24.3
 - ng-particles-demo@1.9.3
 - editor-demo@1.9.3
 - inferno-particles-demo@1.7.3
 - jquery-particles-demo@1.9.3
 - tsparticles-demo@1.9.3
 - preact-particles-demo@1.9.3
 - react-tsparticles-demo@1.9.4
 - svelte-demo@1.7.3
 - particles.vue-demo@1.9.3
 - particles.vue3-demo@1.7.3
 - tsparticles-preset-60fps@1.9.3
 - tsparticles-preset-background-mask@1.9.3
 - tsparticles-preset-basic@1.9.3
 - tsparticles-preset-big-circles@1.9.3
 - tsparticles-preset-bouncing@1.9.3
 - tsparticles-preset-bubbles@1.9.3
 - tsparticles-preset-fire@1.9.3
 - tsparticles-preset-font-awesome@1.9.3
 - tsparticles-preset-fountain@1.9.3
 - tsparticles-preset-sea-anemone@1.9.3
 - tsparticles-preset-snow@1.9.3
 - tsparticles-preset-stars@1.9.3
 - tsparticles-shape-bubble@1.9.3
 - tsparticles-shape-confetti@1.9.3
 - tsparticles-shape-heart@1.9.3
 - tsparticles-shape-multiline-text@1.9.3
 - tsparticles-shape-rounded-rect@1.9.3
 - tsparticles-shape-spiral@1.9.3
 - cra-template-particles-typescript@1.7.4
 - cra-template-particles@1.7.4
 - tsparticles-website@1.24.3

matteobruni committed:
build: updated website with new editor presets

matteobruni committed:
build: added website index confetti preset

matteobruni committed:
build: added website index confetti preset, disabled for issues

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
Merge pull request #1445 from matteobruni/v1

1.24.3

matteobruni committed:
build: fixed confettis on website

matteobruni committed:
build: added share buttons

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
Merge pull request #1447 from matteobruni/v1

website

matteobruni committed:
build: editor updates

matteobruni committed:
chore(release): published new version

 - tsparticles-editor@1.24.4
 - editor-demo@1.9.4
 - tsparticles-website@1.24.4

matteobruni committed:
build: website editor updates

matteobruni committed:
fix: improved shapes with full delta object
fix: fixed confetti with delta object

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
fix: fixed confetti shape

matteobruni committed:
chore(release): published new version

 - ng-particles@2.8.0
 - inferno-particles@1.8.0
 - jquery-particles@1.25.0
 - preact-particles@1.25.0
 - react-tsparticles@1.25.0
 - svelte-particles@1.8.0
 - particles.vue@2.8.0
 - particles.vue3@1.8.0
 - tsparticles-editor@1.25.0
 - tsparticles@1.25.0
 - ng-particles-demo@1.10.0
 - editor-demo@1.10.0
 - inferno-particles-demo@1.8.0
 - jquery-particles-demo@1.10.0
 - tsparticles-demo@1.10.0
 - preact-particles-demo@1.10.0
 - react-tsparticles-demo@1.10.0
 - svelte-demo@1.8.0
 - particles.vue-demo@1.10.0
 - particles.vue3-demo@1.8.0
 - tsparticles-preset-60fps@1.10.0
 - tsparticles-preset-background-mask@1.10.0
 - tsparticles-preset-basic@1.10.0
 - tsparticles-preset-big-circles@1.10.0
 - tsparticles-preset-bouncing@1.10.0
 - tsparticles-preset-bubbles@1.10.0
 - tsparticles-preset-fire@1.10.0
 - tsparticles-preset-font-awesome@1.10.0
 - tsparticles-preset-fountain@1.10.0
 - tsparticles-preset-sea-anemone@1.10.0
 - tsparticles-preset-snow@1.10.0
 - tsparticles-preset-stars@1.10.0
 - tsparticles-shape-bubble@1.10.0
 - tsparticles-shape-confetti@1.10.0
 - tsparticles-shape-heart@1.10.0
 - tsparticles-shape-multiline-text@1.10.0
 - tsparticles-shape-rounded-rect@1.10.0
 - tsparticles-shape-spiral@1.10.0
 - cra-template-particles-typescript@1.8.0
 - cra-template-particles@1.8.0
 - tsparticles-website@1.25.0

matteobruni committed:
build: website updated

matteobruni committed:
build: website update

matteobruni committed:
Merge pull request #1448 from matteobruni/v1

1.25.0

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: fixed merge

ghost commented:
<img src="https://www.deepcode.ai/icons/green_check.svg" width= "50px" align= "left"/> Congratulations :tada:. DeepCode [analyzed](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/759d3e7dbed54dab51828059636f07247453a7b1/matteobruni/tsparticles/4ab6f6e4db9c8df8237f61d88d4e5fe40ff60517/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) your code in 14.104 seconds and we found no issues. Enjoy a moment of no bugs :sunny:.

#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/759d3e7dbed54dab51828059636f07247453a7b1/matteobruni/tsparticles/4ab6f6e4db9c8df8237f61d88d4e5fe40ff60517/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
build: zIndex on confetti shape sample

matteobruni committed:
build: fixed some imports

matteobruni committed:
build: updated some interactors code

dependabot[bot] committed:
build(deps-dev): bump jsoneditor from 9.2.0 to 9.3.0

Bumps [jsoneditor](https://github.com/josdejong/jsoneditor) from 9.2.0 to 9.3.0.
- [Release notes](https://github.com/josdejong/jsoneditor/releases)
- [Changelog](https://github.com/josdejong/jsoneditor/blob/develop/HISTORY.md)
- [Commits](https://github.com/josdejong/jsoneditor/compare/v9.2.0...v9.3.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/core from 7.12.10 to 7.13.14

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.12.10 to 7.13.14.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.13.14/packages/babel-core)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1450 from matteobruni/dependabot/npm_and_yarn/dev/jsoneditor-9.3.0

build(deps-dev): bump jsoneditor from 9.2.0 to 9.3.0

matteobruni committed:
Merge pull request #1451 from matteobruni/dependabot/npm_and_yarn/dev/babel/core-7.13.14

build(deps-dev): bump @babel/core from 7.12.10 to 7.13.14

dependabot[bot] committed:
build(deps-dev): bump @vue/compiler-sfc from 3.0.5 to 3.0.11

Bumps [@vue/compiler-sfc](https://github.com/vuejs/vue-next/tree/HEAD/packages/compiler-sfc) from 3.0.5 to 3.0.11.
- [Release notes](https://github.com/vuejs/vue-next/releases)
- [Changelog](https://github.com/vuejs/vue-next/blob/master/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vue-next/commits/v3.0.11/packages/compiler-sfc)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1452 from matteobruni/dependabot/npm_and_yarn/dev/vue/compiler-sfc-3.0.11

build(deps-dev): bump @vue/compiler-sfc from 3.0.5 to 3.0.11

matteobruni committed:
build: deps updated

matteobruni committed:
feat: particle when destroyed can be splitted, closes #994

matteobruni committed:
build: code improved

matteobruni committed:
build: code imports fixed

matteobruni committed:
build: more code improved

matteobruni committed:
build: updated lint rules

matteobruni committed:
build: restored broken preact/react webpack configuration

matteobruni committed:
build: updates

matteobruni committed:
Merge branch 'v1' into dev

matteobruni committed:
build: fixed merge errors

matteobruni committed:
build: fixed merge errors

matteobruni committed:
build: updates

matteobruni committed:
build: fixed merge and updates

dependabot[bot] committed:
build(deps): bump tslib from 2.1.0 to 2.2.0

Bumps [tslib](https://github.com/Microsoft/tslib) from 2.1.0 to 2.2.0.
- [Release notes](https://github.com/Microsoft/tslib/releases)
- [Commits](https://github.com/Microsoft/tslib/compare/2.1.0...2.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1459 from matteobruni/dependabot/npm_and_yarn/dev/tslib-2.2.0

build(deps): bump tslib from 2.1.0 to 2.2.0

matteobruni committed:
feat: particle init on shape drawers

matteobruni committed:
build: converted to class drawers all external shapes

matteobruni committed:
build: updated spiral and its demo

matteobruni committed:
build: fixed some imports

matteobruni committed:
build: fixed imports

matteobruni committed:
fix: fixed rounded rect shape

dependabot[bot] committed:
build(deps): bump core-js from 3.10.0 to 3.10.1

Bumps [core-js](https://github.com/zloirock/core-js/tree/HEAD/packages/core-js) from 3.10.0 to 3.10.1.
- [Release notes](https://github.com/zloirock/core-js/releases)
- [Changelog](https://github.com/zloirock/core-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/zloirock/core-js/commits/v3.10.1/packages/core-js)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1462 from matteobruni/dependabot/npm_and_yarn/dev/core-js-3.10.1

build(deps): bump core-js from 3.10.0 to 3.10.1

matteobruni committed:
chore(release): published new version

 - ng-particles@2.9.0
 - inferno-particles@1.9.0
 - jquery-particles@1.26.0
 - preact-particles@1.26.0
 - react-tsparticles@1.26.0
 - svelte-particles@1.9.0
 - particles.vue@2.9.0
 - particles.vue3@1.9.0
 - tsparticles-editor@1.26.0
 - tsparticles@1.26.0
 - ng-particles-demo@1.11.0
 - editor-demo@1.11.0
 - inferno-particles-demo@1.9.0
 - jquery-particles-demo@1.11.0
 - tsparticles-demo@1.11.0
 - preact-particles-demo@1.11.0
 - react-tsparticles-demo@1.11.0
 - svelte-demo@1.9.0
 - particles.vue-demo@1.11.0
 - particles.vue3-demo@1.9.0
 - tsparticles-preset-60fps@1.11.0
 - tsparticles-preset-background-mask@1.11.0
 - tsparticles-preset-basic@1.11.0
 - tsparticles-preset-big-circles@1.11.0
 - tsparticles-preset-bouncing@1.11.0
 - tsparticles-preset-bubbles@1.11.0
 - tsparticles-preset-fire@1.11.0
 - tsparticles-preset-font-awesome@1.11.0
 - tsparticles-preset-fountain@1.11.0
 - tsparticles-preset-sea-anemone@1.11.0
 - tsparticles-preset-snow@1.11.0
 - tsparticles-preset-stars@1.11.0
 - tsparticles-shape-bubble@1.11.0
 - tsparticles-shape-confetti@1.11.0
 - tsparticles-shape-heart@1.11.0
 - tsparticles-shape-multiline-text@1.11.0
 - tsparticles-shape-rounded-rect@1.11.0
 - tsparticles-shape-spiral@1.11.0
 - cra-template-particles-typescript@1.9.0
 - cra-template-particles@1.9.0
 - tsparticles-website@1.26.0

matteobruni committed:
Merge pull request #1453 from matteobruni/v1

1.26.0

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: website updated

matteobruni committed:
Merge pull request #1464 from matteobruni/v1

Website updated

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
feat: added new split options to editor

matteobruni committed:
feat: added a first release of riot.js component

matteobruni committed:
build: updated components and core library package description including riot.js

matteobruni committed:
build: riot component now builds when publishing

matteobruni committed:
build: updated riot test

matteobruni committed:
build: fixed shapes error after v1 merge

matteobruni committed:
build: removed schema from project since it's generated

dependabot[bot] committed:
build(deps-dev): bump rollup from 2.44.0 to 2.45.0

Bumps [rollup](https://github.com/rollup/rollup) from 2.44.0 to 2.45.0.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v2.44.0...v2.45.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/plugin-proposal-decorators

Bumps [@babel/plugin-proposal-decorators](https://github.com/babel/babel/tree/HEAD/packages/babel-plugin-proposal-decorators) from 7.13.5 to 7.13.15.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.13.15/packages/babel-plugin-proposal-decorators)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
fix: particles could result misplaced at the beginning, this is now fixed

matteobruni committed:
Merge pull request #1467 from matteobruni/dependabot/npm_and_yarn/dev/rollup-2.45.0

build(deps-dev): bump rollup from 2.44.0 to 2.45.0

matteobruni committed:
Merge pull request #1468 from matteobruni/dependabot/npm_and_yarn/dev/babel/plugin-proposal-decorators-7.13.15

build(deps-dev): bump @babel/plugin-proposal-decorators from 7.13.5 to 7.13.15

matteobruni committed:
build: updated riot component to match all the others

matteobruni committed:
chore(release): published new version

 - ng-particles@2.9.1
 - inferno-particles@1.9.1
 - jquery-particles@1.26.1
 - preact-particles@1.26.1
 - react-tsparticles@1.26.1
 - riot-particles@1.0.0
 - svelte-particles@1.9.1
 - particles.vue@2.9.1
 - particles.vue3@1.9.1
 - tsparticles-editor@1.26.1
 - tsparticles@1.26.1
 - ng-particles-demo@1.11.1
 - editor-demo@1.11.1
 - inferno-particles-demo@1.9.1
 - jquery-particles-demo@1.11.1
 - tsparticles-demo@1.11.1
 - preact-particles-demo@1.11.1
 - react-tsparticles-demo@1.11.1
 - riot-particles-demo@1.0.0
 - svelte-demo@1.9.1
 - particles.vue-demo@1.11.1
 - particles.vue3-demo@1.9.1
 - tsparticles-preset-60fps@1.11.1
 - tsparticles-preset-background-mask@1.11.1
 - tsparticles-preset-basic@1.11.1
 - tsparticles-preset-big-circles@1.11.1
 - tsparticles-preset-bouncing@1.11.1
 - tsparticles-preset-bubbles@1.11.1
 - tsparticles-preset-fire@1.11.1
 - tsparticles-preset-font-awesome@1.11.1
 - tsparticles-preset-fountain@1.11.1
 - tsparticles-preset-sea-anemone@1.11.1
 - tsparticles-preset-snow@1.11.1
 - tsparticles-preset-stars@1.11.1
 - tsparticles-shape-bubble@1.11.1
 - tsparticles-shape-confetti@1.11.1
 - tsparticles-shape-heart@1.11.1
 - tsparticles-shape-multiline-text@1.11.1
 - tsparticles-shape-rounded-rect@1.11.1
 - tsparticles-shape-spiral@1.11.1
 - cra-template-particles-typescript@1.9.1
 - cra-template-particles@1.9.1
 - tsparticles-website@1.26.1

matteobruni committed:
build: preparing riot component for publishing

matteobruni committed:
Merge pull request #1465 from matteobruni/v1

1.26.1

matteobruni committed:
build: website update

matteobruni committed:
build: website update

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
Merge pull request #1469 from matteobruni/v1

website update

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: updates after merge from v1

matteobruni committed:
fix: fixed issue with particles rotated randomically without a rotate config

matteobruni committed:
build: fixed broken test

matteobruni committed:
chore(release): published new version

 - ng-particles@2.9.2
 - inferno-particles@1.9.2
 - jquery-particles@1.26.2
 - preact-particles@1.26.2
 - react-tsparticles@1.26.2
 - riot-particles@1.0.1
 - svelte-particles@1.9.2
 - particles.vue@2.9.2
 - particles.vue3@1.9.2
 - tsparticles-editor@1.26.2
 - tsparticles@1.26.2
 - ng-particles-demo@1.11.2
 - editor-demo@1.11.2
 - inferno-particles-demo@1.9.2
 - jquery-particles-demo@1.11.2
 - tsparticles-demo@1.11.2
 - preact-particles-demo@1.11.2
 - react-tsparticles-demo@1.11.2
 - riot-particles-demo@1.0.1
 - svelte-demo@1.9.2
 - particles.vue-demo@1.11.2
 - particles.vue3-demo@1.9.2
 - tsparticles-preset-60fps@1.11.2
 - tsparticles-preset-background-mask@1.11.2
 - tsparticles-preset-basic@1.11.2
 - tsparticles-preset-big-circles@1.11.2
 - tsparticles-preset-bouncing@1.11.2
 - tsparticles-preset-bubbles@1.11.2
 - tsparticles-preset-fire@1.11.2
 - tsparticles-preset-font-awesome@1.11.2
 - tsparticles-preset-fountain@1.11.2
 - tsparticles-preset-sea-anemone@1.11.2
 - tsparticles-preset-snow@1.11.2
 - tsparticles-preset-stars@1.11.2
 - tsparticles-shape-bubble@1.11.2
 - tsparticles-shape-confetti@1.11.2
 - tsparticles-shape-heart@1.11.2
 - tsparticles-shape-multiline-text@1.11.2
 - tsparticles-shape-rounded-rect@1.11.2
 - tsparticles-shape-spiral@1.11.2
 - cra-template-particles-typescript@1.9.2
 - cra-template-particles@1.9.2
 - tsparticles-website@1.26.2

matteobruni committed:
build: website updated

matteobruni committed:
build: updated react/preact/inferno d.ts file, closes #1472

matteobruni committed:
chore(release): published new version

 - inferno-particles@1.9.3
 - preact-particles@1.26.3
 - react-tsparticles@1.26.3
 - inferno-particles-demo@1.9.3
 - preact-particles-demo@1.11.3
 - react-tsparticles-demo@1.11.3
 - cra-template-particles-typescript@1.9.3
 - cra-template-particles@1.9.3
 - tsparticles-website@1.26.3

matteobruni committed:
Merge pull request #1471 from matteobruni/v1

1.26.2

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: fixed riot demo deps

matteobruni committed:
build: updated website version

matteobruni committed:
chore(release): published new version

 - tsparticles@2.0.0-alpha.7
 - tsparticles-slim@2.0.0-alpha.7
 - ng-particles@3.0.0-alpha.7
 - inferno-particles@2.0.0-alpha.7
 - jquery-particles@2.0.0-alpha.7
 - preact-particles@2.0.0-alpha.7
 - react-tsparticles@2.0.0-alpha.7
 - riot-particles@2.0.0-alpha.7
 - svelte-particles@2.0.0-alpha.7
 - particles.vue@3.0.0-alpha.7
 - particles.vue3@2.0.0-alpha.7
 - ng-particles-demo@2.0.0-alpha.7
 - editor-demo@2.0.0-alpha.7
 - inferno-particles-demo@2.0.0-alpha.7
 - jquery-particles-demo@2.0.0-alpha.7
 - preact-particles-demo@2.0.0-alpha.7
 - react-tsparticles-demo@2.0.0-alpha.7
 - riot-particles-demo@2.0.0-alpha.7
 - svelte-demo@2.0.0-alpha.7
 - tsparticles-demo@2.0.0-alpha.7
 - particles.vue-demo@2.0.0-alpha.7
 - particles.vue3-demo@2.0.0-alpha.7
 - tsparticles-editor@2.0.0-alpha.7
 - tsparticles-engine@2.0.0-alpha.7
 - tsparticles-interaction-light@2.0.0-alpha.7
 - tsparticles-interaction-external-attract@2.0.0-alpha.7
 - tsparticles-interaction-external-bounce@2.0.0-alpha.7
 - tsparticles-interaction-external-bubble@2.0.0-alpha.7
 - tsparticles-interaction-external-connect@2.0.0-alpha.7
 - tsparticles-interaction-external-grab@2.0.0-alpha.7
 - tsparticles-interaction-external-repulse@2.0.0-alpha.7
 - tsparticles-interaction-external-trail@2.0.0-alpha.7
 - tsparticles-interaction-particles-attract@2.0.0-alpha.7
 - tsparticles-interaction-particles-collisions@2.0.0-alpha.7
 - tsparticles-interaction-particles-links@2.0.0-alpha.7
 - tsparticles-interaction-particles-move@2.0.0-alpha.7
 - tsparticles-interaction-particles-parallax@2.0.0-alpha.7
 - tsparticles-interaction-particles-repulse@2.0.0-alpha.7
 - tsparticles-plugin-absorbers@2.0.0-alpha.7
 - tsparticles-plugin-emitters@2.0.0-alpha.7
 - tsparticles-plugin-infection@2.0.0-alpha.7
 - tsparticles-plugin-polygon-mask@2.0.0-alpha.7
 - tsparticles-preset-big-circles@2.0.0-alpha.7
 - tsparticles-preset-bubbles@2.0.0-alpha.7
 - tsparticles-preset-fire@2.0.0-alpha.7
 - tsparticles-preset-fountain@2.0.0-alpha.7
 - tsparticles-preset-links@2.0.0-alpha.7
 - tsparticles-preset-sea-anemone@2.0.0-alpha.7
 - tsparticles-preset-snow@2.0.0-alpha.7
 - tsparticles-preset-stars@2.0.0-alpha.7
 - tsparticles-shape-bubble@2.0.0-alpha.7
 - tsparticles-shape-circle@2.0.0-alpha.7
 - tsparticles-shape-confetti@2.0.0-alpha.7
 - tsparticles-shape-heart@2.0.0-alpha.7
 - tsparticles-shape-image@2.0.0-alpha.7
 - tsparticles-shape-line@2.0.0-alpha.7
 - tsparticles-shape-multiline-text@2.0.0-alpha.7
 - tsparticles-shape-polygon@2.0.0-alpha.7
 - tsparticles-shape-rounded-rect@2.0.0-alpha.7
 - tsparticles-shape-spiral@2.0.0-alpha.7
 - tsparticles-shape-square@2.0.0-alpha.7
 - tsparticles-shape-star@2.0.0-alpha.7
 - tsparticles-shape-text@2.0.0-alpha.7
 - cra-template-particles-typescript@2.0.0-alpha.7
 - cra-template-particles@2.0.0-alpha.7
 - tsparticles-updater-angle@2.0.0-alpha.7
 - tsparticles-updater-color@2.0.0-alpha.7
 - tsparticles-updater-life@2.0.0-alpha.7
 - tsparticles-updater-opacity@2.0.0-alpha.7
 - tsparticles-updater-orbit@2.0.0-alpha.7
 - tsparticles-updater-out-modes@2.0.0-alpha.7
 - tsparticles-updater-size@2.0.0-alpha.7
 - tsparticles-updater-stroke-color@2.0.0-alpha.7
 - tsparticles-website@2.0.0-alpha.7

matteobruni committed:
build: pre release updates

matteobruni committed:
build: pre-build commit

matteobruni committed:
build: fixed particles startup position

dependabot[bot] committed:
build(deps-dev): bump uglify-js from 3.13.3 to 3.13.4

Bumps [uglify-js](https://github.com/mishoo/UglifyJS) from 3.13.3 to 3.13.4.
- [Release notes](https://github.com/mishoo/UglifyJS/releases)
- [Commits](https://github.com/mishoo/UglifyJS/compare/v3.13.3...v3.13.4)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1474 from matteobruni/dependabot/npm_and_yarn/dev/uglify-js-3.13.4

build(deps-dev): bump uglify-js from 3.13.3 to 3.13.4

dependabot[bot] committed:
build(deps-dev): bump css-loader from 5.2.0 to 5.2.1

Bumps [css-loader](https://github.com/webpack-contrib/css-loader) from 5.2.0 to 5.2.1.
- [Release notes](https://github.com/webpack-contrib/css-loader/releases)
- [Changelog](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/css-loader/compare/v5.2.0...v5.2.1)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1475 from matteobruni/dependabot/npm_and_yarn/dev/css-loader-5.2.1

build(deps-dev): bump css-loader from 5.2.0 to 5.2.1
